css/style.css

```diff
@@ -0,0 +1,112 @@
+
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #f5f5f5;
+    margin: 0;
+}
+
+.ticket{
+    box-sizing: border-box;
+    width: 782px;
+    height: 273px;
+    background-color: #efc684;
+    border-radius: 10px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-family: Arial, sans-serif;
```

Comment:
Please find the correct font in figma 
![image](https://github.com/user-attachments/assets/27aad601-d9eb-42e7-b808-c32b449017f6)